routers/router.go

```diff
@@ -57,8 +57,10 @@ func initAPI() {
 	beego.Router("/api/get-sessions", &controllers.ApiController{}, "GET:GetSessions")
 	beego.Router("/api/get-session", &controllers.ApiController{}, "GET:GetConnSession")
 	beego.Router("/api/update-session", &controllers.ApiController{}, "POST:UpdateSession")
-	beego.Router("/api/add-session", &controllers.ApiController{}, "POST:AddSession")
+	beego.Router("/api/create-session", &controllers.ApiController{}, "POST:AddSession")
```

Comment:
Better keep using `add-session` to be aligned with other APIs

The other one can be named as `addSessionInternal`